src/components/BreedsList.vue

```diff
@@ -226,6 +231,20 @@ export default {
     //     });
     //   }
     // },
+    scrollToView(index) {
+      console.log(index);
+      this.displayedBreeds.forEach((breed) => {
+        if (breed.isActive) {
+          console.log(breed);
+          this.$refs.resultListContainer.children[
+            this.lastBreedState[index]
+          ].scrollIntoView({
+            behavior: "smooth",
+            block: "center",
+          });
+        }
+      });
+    },
```

Comment:
Namesto da watch-am computed `displayedBreeds`, ker ni optimalno, sem sedaj v TheNavigation componenti, emitov event  `countdown` in `addingUp` za `Back & Forward btn` za `index `data property, s katero beležim katera kartica je bila klikana  in jo scrolla na sredino, in pa dodal sem watcher  `indexClicked ` v `BreedsList `komponenti, ki watcha index klikane kartice in scrolla na sredino za vsako odprto kartico.